src/webpack.js

```diff
@@ -1,25 +1,58 @@
+const path = require('path')
+const Promise = require('bluebird')
+const fs = Promise.promisifyAll(require('fs'))
 const webpack = require('webpack')
-const WebpackDevServer = require('webpack-dev-server')
+const webpackMiddleware = require('webpack-dev-middleware')
 const config = require('./webpack.config.js')
+const { COMPONENT_ABSOLUTE_PATH, PORT } = require('./constants')
 
-const WEBPACK_PORT = 8080
+const getModules = async () => {
```

Comment:
Should we extract this function into it's own js file ?
because this part will be more complicated in the future...